src/state/state_manager.py

```diff
@@ -1,12 +1,7 @@
 class StateManager:
     def __init__(self):
         self.last_message = None
-
-    def set_last_message(self, message):
-        self.last_message = message
-
-    def get_last_message(self):
-        return self.last_message
+        self.canvas = None
```

Comment:
I like the refactor here, just make sure to update the places where `set_last_message()` and `get_last_message()` are used

Reply:
truuu
done